libvast/fbs/partition.fbs

```diff
@@ -42,9 +42,10 @@ table LegacyPartition {
   /// The available layouts in this partition, flattened with the layout name
   /// embedded in the field names, pruned of attributes, and serialized using
   /// CAF 0.17's binary serializer.
-  combined_layout: [ubyte];
+  combined_layout_caf_0_17: [ubyte];
```

Comment:
Please add a comment that we can mark this with a `(deprecated)` attribute once the minimum partition version is 3. The current version is 2, so for partitions that have version 3 we'll know that they cannot have this field set, so once we bumped the minimum to 3 we can remove the code generation for this field.

The same applies to the `combined_layout` field.